pkg/grpc/marshaler.go

```diff
@@ -29,16 +28,27 @@ func (c customMarshaler) Unmarshal(data []byte, v interface{}) error {
 		return unmarshalError
 	}
 	log.Warnf("DEPRECATED: The duration format only supports seconds: %q", unmarshalError)
-	messageV1 := protoadapt.MessageV1Of(suppressCVERequest)
-	err := jsonpb.Unmarshal(bytes.NewBuffer(data), messageV1)
+	goVersion := SuppressCVERequestGo{}
+	err := json.Unmarshal(data, &goVersion)
 	if err != nil {
+		log.Warnf(err.Error())
 		// We want users choose the new format.
 		return unmarshalError
 	}
-	v = protoadapt.MessageV2Of(messageV1)
+	suppressCVERequest.Cves = goVersion.CVES
+	duration, err := time.ParseDuration(goVersion.Duration)
+	if err != nil {
+		return unmarshalError
+	}
+	suppressCVERequest.Duration = protocompat.DurationProto(duration)
 	return nil
 }
 
+type SuppressCVERequestGo struct {
+	CVES     []string
+	Duration string
+}
```

Comment:
While I don't consider them as absolutely mandatory, it might be good to add JSON annotations to the structure fields in order to help the decoder work.